src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -9,19 +9,19 @@
 import seedu.address.logic.Messages;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.person.Person;
 
 /**
  * Deletes a person identified using it's displayed index from the address book.
  */
-public class DeleteCommand extends Command {
+public abstract class DeleteCommand<T> extends Command {
 
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + ": Deletes the entity identified by subcommand 'contact', 'company' or 'job' "
+            + "the index number used in the displayed person list.\n"
+            + "Parameters: [contact/job/company] INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " contact  1";
```

Comment:
Nit 
- Should append 'and' to the first line
- In the last line, there is double white-space between 'contact' and '1'

Reply:
got it